docs/sources/fundamentals/dashboards-overview/index.md

```diff
@@ -0,0 +1,88 @@
+---
+aliases:
+  - ../basics/timeseries/
+description: Grafana Dashboards Overview
+keywords:
+  - grafana
+  - dashboards
+  - panel
+  - data source
+  - transform
+  - query
+title: Grafana Dashboards Overview
+weight: 300
+---
+
+# Grafana Dashboards Overview
+
+What is a dashboard? This term is used widely in the observability space, but what does it actually mean? The term is taken from automobiles where a dashboard provides access to the controls a driver uses to operate a vehicle. The same applies to digital dashboards, except that digital dashboards help us understand and drive systems. The purpose of this topic is to explain how Grafana dashboards work to make it easier for you to build your own.
+
+The following image illustrates a sample Grafana dashboard.
+
+{{< figure src="/media/docs/grafana/dashboards-overview/complex-dashboard-example.png" max-width="750px" caption="Example Grafana Dashboard" >}}
+
+A Grafana dashboard is a collection of data visualization panels. Each panel relies on a collection of components that  takes raw facts from a data source and shapes them into a beautiful visualization. The raw facts pass through three gates before being visualized, one of which is optional: a plugin, a query, and a transform.
+
+The following image illustrates each of the gates, and the following sections describe each element, how it’s used, and why it’s necessary. The journey of a visualization begins at the data source and works its way through to a Grafana dashboard.
+
+{{< figure src="/media/docs/grafana/dashboards-overview/dashboard-component-architecture.png" max-width="750px" caption="Dashboard component architecture" >}}
+
+## Data Source
+
+A data source is any entity that contains data. A data source can be a SQL database, Grafana Loki, Grafana Mimir, or a JSON-based API. A data source can even be a flat CSV file. A dashboard visualization starts with the data source that contains the data you want to see.
+
+Understanding the differences between various data sources can be a challenge because each data source has a unique structure and requires different query methods. To simplify this complexity, Grafana Labs has developed plugins that unify these data sources into one coherent view.
+
+## Plugins
+
+A Grafana plugin is software that adds new capabilities to Grafana. They come in many types, but right now we will address _data source plugins_. The job of a Grafana data source plugin is to take a query you want answered, retrieve the  data from the data source, and reconcile the differences in data models using a unified data structure called a [data frame](https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/). The data coming into the plugin from the data source might be many different formats (such as JSON, rows and columns, or CSV), but when it leaves the plugin and moves through the rest of the gates toward a visualization, it is always data frames.
```

Comment:
@moxious , so there are data source plugins - what other types of plugins are there? And it occurs to me that we can visualize data via intergrations, which does not rely on plugins. Do we address integrations in this topic?

Reply:
disagree.  Trying to keep it simple.  The proliferation of different types of plugins and integrations is a mess, and draws us off of the central topic of "how do dashboards work".  I agree we should explain this, but don't think here is the right place to do it.  It'll turn into a big digression that waters down the point I'm trying to make about how data gets in.